### DISCUSSION TITLE
Para COMEÇAR com Front-End, HTML, CSS e JAVASCRIPT, seria suficiente?

### DISCUSSION THREAD

**Discussion by manoelrobertok** [frontendbr/forum#2413] [Dicas]
Estou tendo muita **dificuldade com React js**.Embora, me dar melhor com Html, Css e JavaScript, não posso afirmar que já tenho domínio.
 **Html, Css e JavaScript,** seria o suficiente para entrar no mercado de trabalho? Valeu pessoal!

**Comment by ninetails** [comment]
Suficiente em teoria seria.

Porém lembre que essas são as skills básicas de frontend. Caso você não tenha o diferencial ou outros diferenciais, outro candidato a mesma vaga terá.

Falar que só HTML, CSS e JavaScript seriam suficientes para alguma vaga que não peça framework pode até ser suficiente, mas lembre que para uma vaga não é só considerado o teu conhecimento, também são considerados os conhecimentos de seus concorrentes. Quanto mais diferenciais, mais você vai sobressair num processo seletivo.

Ou por outro lado, quanto menos requisitos você cumprir nas vagas, menor vai ser o seu valor de mercado também.

Se quiser e tiver essa emergência em entrar no mercado de trabalho com estes requisitos, recomendo tentar procurar vagas em agências de publicidade.

**Reply by josekrauss020** [reply]
Por que especificamente em agências de publicidade?

**Reply by ninetails** [reply]
Mencionei agencias de publicidade e produtoras mas mais num sentido de locais onde há uma alta demanda pra fazer sites para marketing. Locais que procuram focar na alta produção de páginas com pouco reaproveitamento de código e que dificilmente pensam em evoluir um produto e que acabam focando na entrega mais que num produto em si.

Não dizendo que estas não possuem frameworks e boas práticas, nem que todas são assim por via de regra. Ou que não usem frameworks. Mas provavelmente irão exigir mais do candidato que entregue com rapidez e aprenda na empresa.

---

Digo, pensando melhor na minha colocação de meses atrás pode não ter sido das melhores, essa era uma visão bem antiga de mercado. Hoje pode ser que não sejam mais assim, mas antigamente era bem comum ter essa visão.

**Comment by vitocaio** [comment]
Opa beleza man? 
Não se limite pelas dificuldades, tente criar novos métodos de aprendizagem, é importante entender que em Tech vai existir gaps, mas é de grande importância supera-los

**Comment by viralatron** [comment]
> Estou tendo muita **dificuldade com React js**.Embora, me dar melhor com Html, Css e JavaScript, não posso afirmar que já tenho domínio. **Html, Css e JavaScript,** seria o suficiente para entrar no mercado de trabalho? Valeu pessoal!

cara, é o que a ninetails disse. só adiciono uma coisa: se estiver mais dificuldade de entender o React, procure outros materiais. sempre bom diversificar suas fontes de conteúdo e tentar seu melhor para entender a teoria.
